src/js/chat.js

```diff
@@ -505,7 +501,7 @@ sock.onmessage = function (e) {
             case "332":
                 // Topic, display?
                 break;
-            // Part and quit both need to be handled the same way in our case - a user left the room
+                // Part and quit both need to be handled the same way in our case - a user left the room
```

Comment:
These indents should not be changed - they're referring to the next block, not the previous one. I'd be ok with moving them INSIDE the following block and indenting though.

I.E.:
```
case "KICK":
    // Check if user...
```